nixos/modules/config/swap.nix

```diff
@@ -185,6 +185,7 @@ in
           { description = "Initialisation of swap device ${sw.device}";
             wantedBy = [ "${realDevice'}.swap" ];
             before = [ "${realDevice'}.swap" ];
+            after = mkIf (config.security.rngd.enable && sw.randomEncryption.enable) [ "rngd.service" ];
```

Comment:
I think a comment mentioning the entropy starvation here would also be helpful.

Reply:
Added some more comments to my commits